extensions/amp-date-picker/0.1/test/validator-amp-date-picker.html

```diff
@@ -102,6 +102,12 @@
   <!-- Valid: range amp-date-picker with minimum nights -->
   <amp-date-picker type="range" layout="fixed-height" height="360" minimum-nights="0">
   </amp-date-picker>
+  <!-- Valid: single amp-date-picker with date attribute -->
+  <amp-date-picker layout="fixed-height" height="360" date="2018-01-01">
+  </amp-date-picker>
+  <!-- Valid: range amp-date-picker with start-date and end-date attributes -->
```

Comment:
This is invalid, not valid. Error states: `The specified layout 'FIXED_HEIGHT' is not supported by tag 'amp-date-picker[type=range][mode=overlay]'`

Reply:
I don't understand how it's invalid if the validator tests pass. Could you help me understand? I didn't see an error

Reply:
Take a look at the `.out` file and it shows an error for this test. If the `.out` files are updated and not verified that what was updated was intended then the validator test will pass.

Reply:
Oh I see, it's missing `type="range"`